ekm-over-vpc-onboarding/modules/ekm_resources_creation/network.tf

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#Network & Subnet creation
+module "vpc-network" {
+  source       = "terraform-google-modules/network/google"
+  version      = "~> 8.0"
+  project_id   = data.google_project.vpc_project.number
+  network_name = var.network_name
+  mtu          = 1460
+  routing_mode = "REGIONAL"
+
+  subnets = [
+    {
+      subnet_name   = "${var.network_name}-subnet"
+      subnet_region = var.location
+      subnet_ip     = var.subnet_ip_cidr_range
+    }
+  ]
+}
+
+# GDU : 35.199.192.0/19
+# For TPC {PRPTST: 34.116.26.128/27, THP000: 177.222.82.0/25, THQ000: 31.177.4.128/27, GDUTST: 35.190.248.0/22}
```

Comment:
Are these all public? Just making sure before we merge.

Reply:
Good point. Let me remove this.